appcenter-post-clone.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Is moving the file necessary for RN builds? 
If yes we should consider a way of flagging platform types to not do the extra work if not needed.

Reply:
Moving the files, yes. Flags for Platform type, AppCenter does not provide.